recipes/stir/meta.yaml

```diff
@@ -0,0 +1,53 @@
+
+{% set name = "STIR" %}
+{% set version = "v3_0_1" %}
+{% set sha256 = "4f257fa7a4b15c9d6eddf981d80b79fc27a3ed10df4b8184d507d82a22f3596e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/olivierverdier/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [py2k]
+  skip: true  # [win and py<36]
+  number: 0
+
+requirements:
+  build:
+    - python
+    - cmake
+    - boost
```

Comment:
So this is actually the package that includes `boost::python`, and it's pulling in numpy, which [is actually required](https://github.com/UCL/STIR/blob/master/src/swig/CMakeLists.txt#L40) and is currently being linked against. I think you probably just want `boost-cpp 1.66.0`, and need to do `numpy 1.8.*` as a `build` req and `numpy >=1.8` as a `run` req. Right now it's pulling a newer numpy and could break if someone tried to use it with an older one.

Reply:
Ok, I'll try that.

Reply:
This particular combination does not work on Windows.

I get on `Minconda36`:
```
The following specifications were found to be in conflict:
  - numpy 1.8.* -> python 2.6* -> vc 9
  - python 3.6*
```
and on `Miniconda-x64`:
```
The following specifications were found to be in conflict:
  - numpy 1.8.* -> python 3.4* -> vc 10
  - python 3.6*
```

Reply:
Never mind, this seems to be fixed now.